src/main/proto/ga4gh/bio_metadata.proto

```diff
@@ -134,7 +140,7 @@ message BioCharacteristic {
   //     termId: "UBERON:0010033",
   //     term: "posterior part of tongue",
   //
-  repeated OntologyTerm ontologyTerms = 2;
+  repeated OntologyTerm ontologyTerms = 3;
 
```

Comment:
We shouldn't renumber pre-existing fields in protobuf. Add a new number instead.